sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -39,6 +41,11 @@
 @Experimental
 public interface SupportsNamespaces extends CatalogPlugin {
 
+  String NAMESPACE_PROP_LOCATION = "location";
```

Comment:
shall we also put the table properties in `TableCatalog` instead of `Table`?

Reply:
yea, updated.